api/proto/teleport/resourceusage/v1/device_trust.proto

```diff
@@ -0,0 +1,31 @@
+// Copyright 2023 Gravitational, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package teleport.resourceusage.v1;
+
+option go_package = "github.com/gravitational/teleport/api/gen/proto/go/teleport/resourceusage/v1;resourceusagev1";
+
+// DevicesUsage holds aggregated information about trusted device usage.
+message DevicesUsage {
+  // Devices usage limit.
+  // Always zero if the account usage type is UNLIMITED.
```

Comment:
nit: the full name makes it easier to find the relevant piece of code being referred to (same for below)
```suggestion
  // Always zero if the account has ACCOUNT_USAGE_TYPE_UNLIMITED.
```

Reply:
Done.